test/js/reducers/questionnaire.spec.js

```diff
@@ -817,7 +818,7 @@ describe('questionnaire reducer', () => {
 
   describe('helpers', () => {
     it('should provide valid answers for multiple-choice steps', () => {
-      const questionnaire = deepFreeze({
+      const questionnaire: Questionnaire = deepFreeze({
```

Comment:
Either remove this type annotation or declare an intermediate variable with the literal object and apply `deepFreeze` to that. Otherwise this gives a false impression of type safety (because `deepFreeze` will type check no matter what you pass to it).